definitions/liferay-workflow-definition_7_3_0.xsd

```diff
@@ -451,6 +451,7 @@
 	</xs:simpleType>
 	<xs:simpleType name="role-type">
 		<xs:restriction base="xs:string">
+			<xs:enumeration value="account" />
```

Comment:
This is temporary change. We'll need to create a new schema `liferay-workflow-definition_7_4_0.xsd` and apply the change.